elements/lisk-cryptography/src/buffer.ts

```diff
@@ -21,6 +21,7 @@ export const intToBuffer = (
 	value: number | string,
 	byteLength: number,
 	endianness: string = BIG_ENDIAN,
+	signed: boolean = false,
```

Comment:
**Not** for this PR but for future tech-debt tasks maybe we should consider having to functions `intToBigEndianBuffer` and `intToLittleEndianBuffer` (though I think we might be moving to only big endianness in which case this comment and any other effort will not be needed) Just for us to think about it as the function is getting too "branchy"

Reply:
yes definitely. I was thinking to refactor but it affects lot of places, and we gonna change with generic serialization anyway, so that's why i kept it for now